sdk/src/main/java/com/mapbox/maps/MapboxConstants.kt

```diff
@@ -7,7 +7,7 @@ import java.util.*
 /**
  * Name of the database file.
  */
-const val DATABASE_NAME = "mbx.db"
+const val DATABASE_NAME = "ambient_cache.db"
```

Comment:
I think ios is also using a mapbox directory to contain this db file.

Reply:
The folder originally was `<app id>/.mapbox/cache.db` - PR changes it to `com.mapbox.maps/ambient_cache.db` (since we're in the apps support directory)

Reply:
Agree with @tobrun here, I'd suggest to use /mapbox/ambient_cache.db